### ISSUE TITLE
Shard block vs. beacon block

### ISSUE THREAD

**Issue by paulhauner** [ethereum/eth2.0-specs#66]
I've noticed that there's an assumption that "block" means a beacon block and "shard block" means a shard block.

For example:
```python
    # Block hashes not part of the current chain, oldest to newest
    'oblique_parent_hashes': ['hash32'],
    # Shard block hash being attested to
    'shard_block_hash': 'hash32',
```

I think it would be useful (particularly to new readers) to either specify "beacon block" or to formally establish the "block == beacon block" rule.

**Comment by JustinDrake** [comment]
We use to call shard blocks "collations" but that was also confusing nomenclature. Agreed that we should be explicit about the type of block everywhere.

**Comment by djrtwo** [comment]
agreed

**Comment by djrtwo** [comment]
closed via #87
